.github/workflows/test-appetize.yml

```diff
@@ -1,83 +1,46 @@
 # Manually trigger an android build and deploy to Appetize.io
 
 name: Test Appetize
+concurrency:
+  group: test-appetize-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 on:
   pull_request:
     types: [labeled, synchronize]
     branches:
       - deployment/*
       - master
 jobs:
-  # Set the deployment name, to be used as the name of the environment in which to run the following job
-  # Pattern found in this question: https://github.com/actions/runner/issues/998#issue-817330769
-  set_deployment_name:
+  build:
     if: contains(github.event.pull_request.labels.*.name, 'Test - appetize')
-    outputs:
-      DEPLOYMENT_NAME: ${{ steps.get_deployment_name.outputs.DEPLOYMENT_NAME }}
-    runs-on: ubuntu-latest
-    steps:
-    # Extract deployment name as the slug of target PR branch via string operators, e.g. /deployment/plh_tz -> plh_tz
-    # https://www.linuxjournal.com/article/8919, https://github.com/greenbone/gsa/actions/runs/1277581365/workflow
-    - name: Get deployment name
-      id: get_deployment_name
-      run: |
-        if [ "$GITHUB_BASE_REF" = "master" ];                    \
-        then echo "DEPLOYMENT_NAME=plh_global" >> $GITHUB_OUTPUT;              \
-        else echo "DEPLOYMENT_NAME=${GITHUB_BASE_REF##*/}" >> $GITHUB_OUTPUT;  \
-        fi;
+    uses: ./.github/workflows/android-build.yml
+    secrets: inherit
+    with:
+      # use branch name from PR target as default
+      # https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+      deployment: ${{ github.base_ref }} 
+      debug-apk: true
+      release-bundle: false
+   
   test_appetize:
-    needs: set_deployment_name
-    if: contains(github.event.pull_request.labels.*.name, 'Test - appetize')
+    needs: build
     runs-on: ubuntu-latest
-    environment: 
-      name: ${{ needs.set_deployment_name.outputs.DEPLOYMENT_NAME }}
```

Comment:
Previously we were setting the github environment to match the deployment name in order to use the relevant appetize secrets for each deployment, configured [here](https://github.com/IDEMSInternational/parenting-app-ui/settings/environments). As in #1864. 

Are these deployment-dependent `APPETIZE_TOKEN`s still supported? Or is the thought that this won't be needed once deployments exist outside of the code repo?

Reply:
nice catch! I think I naively saw `environment` next to `env` and thought it was a legacy/typo issue, had forgotten all about the deployment-dependent secret environments you set up.

Have reverted and added an additional comment accordingly